### PR TITLE
refactor: qr scanner modal safari camera permissions

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
Close/cancel scanning `X` button doesn't work in Android.

**Review by martinkaintas — APPROVED** [review]

File: src/popup/components/Modals/QrCodeReader.vue [hunk 1]
```diff
@@ -1,11 +1,15 @@
 <template>
+  <!--
+    This modal is displayed only on web and extension.
```
[the remark this answers is not in the document]
Comment (peronczyk) [reply src/popup/components/Modals/QrCodeReader.vue]: @martinkaintas please see my updated comment. I wanted to clearly describe the difference between the mobile and web use case.

**Comment by peronczyk** [comment]
> Close/cancel scanning `X` button doesn't work in Android.

Fixed. Can you confirm?

**Comment by Liubov-crypto** [comment]
LGTM

**Pull request merged**